validationTemplate/src/ModelCalibrationTemplate/CalibrationTemplate.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Aug  4 09:33:34 2021
+
+@author: GARRM
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+# standard library
+import os
+import sys
+# external libraries
+# RAVEN libraries
+## caution: using abspath here does not do desirable things on Windows Mingw.
+import _utils as hutils
+raven_path = os.path.abspath(hutils.get_raven_loc()) 
```

Comment:
FYI `hutils` in `HERON` is a portmanteau for `heron utils`; this doesn't necessarily need changing, just a note.